sdk/purview/azure-analytics-purview-datamap/README.md

```diff
@@ -0,0 +1,65 @@
+# Azure PurviewDataMap client library for Java
+
+Azure PurviewDataMap client library for Java.
+
+This package contains Microsoft Azure PurviewDataMap client library.
+
+## Documentation
+
+Various documentation is available to help you get started
+
+- [API reference documentation][docs]
+- [Product documentation][product_documentation]
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure:azure-analytics-purview-datamap;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-analytics-purview-datamap</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Authentication
+
+[Azure Identity][azure_identity] package provides the default implementation for authenticating the client.
+
+## Key concepts
+
+## Examples
+
+```java com.azure.analytics.purview.datamap.readme
+```
+
+## Troubleshooting
```

Comment:
Add contents.

Reply:
updated

Reply:
Still don't see the content of 'Troubleshooting'. Where did you update at?